package.json

```
@@ -30,5 +29,8 @@
     "mongodb-js-precommit": "^2.0.0",
     "sinon": "^9.0.0"
   },
+  "peerDependencies": {
+    "bson": "^4 || ^5"
```

Can we keep the same minimum version for v4 here?

```suggestion
    "bson": "^4.6.1 || ^5"
```

I’m not sure what the actual minimum is in practice, but this at least preserves some value of “this package worked at some point with that version”

---

Done.